source/lib.civet

```diff
@@ -131,26 +144,18 @@ function updateParentPointers(node: ASTNode, parent?: ASTNodeBase, depth = 1): v
 }
 
 function addPostfixStatement(statement: StatementTuple, ws: ASTNode, post)
-  let children, expressions
-
-  prefix := post.blockPrefix or post.condition?.expression.blockPrefix
-
-  if prefix?.length
-    indent := prefix[0][0]
-    expressions = [...prefix, [indent, statement]]
-    children = [" {\n", ...expressions, "\n", indent?.slice?.(0, -2), "}"]
-  else
-    expressions = [["", statement]]
-    children = [" { ", ...expressions, " }"]
+  expressions := [
+    ...post.blockPrefix or []
```

Comment:
I wonder about the loss of `post.condition?.expression.blockPrefix` here... but maybe it's not the cause of the bug below.

Reply:
The condition expression block prefix is now added in a more uniform way inside `processDeclarationConditions`, this one here is to pick up the random prefixes from coffee style for loops until we move them later on in the parse tree processing.